core/src/client/async_client/mod.rs

```diff
@@ -40,6 +40,8 @@ use tracing::instrument;
 
 use super::{generate_batch_id_range, FrontToBack, IdKind, RequestIdManager};
 
+const LOG_TARGET: &str = "jsonrpsee-client";
```

Comment:
that's fair but I guess I would need to remove the `tracing helper module then` because it will just use the `jsonrpsee-client` so simplicity it could any of these :)

Reply:
Ahh I see, we could keep the `LOG_TARGET` as is for now. And if we ever find things hard to debug and we'd want more granularity, we can come back with a followup 👍

Reply:
I would like to remove the tracing module anyway but let's do a follow up PR :)